RubeusCore/Source/UtilityComponents/loader_component.cpp

```diff
@@ -67,7 +67,7 @@ namespace Rubeus
 
 			ASSERT(std::string("Loading image: ") + std::string(path));
 
-			DevILCall(ilLoadImage(path));
+			DevILCall(ilLoadImage((ILstring) path));
```

Comment:
DevIL had a little bug in its later versions (we were using 1.8) so downgrading to 1.6.7 seems to have fixed it